library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -267,14 +153,14 @@ public void setItems(List<Item> items)
         return mButtonClicks;
     }
 
-    public Observable<Item> iconClicks()
+    public Observable<Pair<Item, Integer>> iconClicks()
     {
-        return mIconClicks;
+        return mListViewCardAdapter.iconClicks();
     }
 
-    public Observable<Item> itemClicks()
+    public Observable<Pair<Item, Integer>> itemClicks()
```

Comment:
Just a heads up on why I made this change even though we aren't taking advantage of it currently. This gives us a handle on the adapter position of the items that was clicked. This can allow for us to edit that specific item within the adapter without keeping a reference to the unchanged Item and more accurately notify the adapter of the item changed.